ui/admin/app/components/form/target/details/index.hbs

```diff
@@ -43,33 +43,34 @@
 
   {{#if (and @model.isNew (feature-flag 'ssh-target'))}}
     <Hds::Form::RadioCard::Group @name='type' @alignment='center' as |G|>
-      <G.Legend>{{t 'form.type.label'}}</G.Legend>
-      {{#each-in this.mapResourceTypeWithIcon as |targetType icon|}}
+      <G.Legend>{{t 'resources.target.form.type.label'}}</G.Legend>
+      <G.HelperText>{{t 'resources.target.form.type.help'}}</G.HelperText>
+      {{#each this.typeMetas as |type|}}
         <G.RadioCard
-          @name={{targetType}}
-          @value={{targetType}}
+          @value={{type.type}}
           @maxWidth='20rem'
-          @checked={{eq targetType @model.type}}
-          {{on 'input' (fn @changeType targetType)}}
+          @checked={{eq type.type @model.type}}
+          {{on 'input' (fn @changeType type.type)}}
           as |R|
         >
-          <R.Label>{{t (concat 'resources.target.types.' targetType)}}</R.Label>
+          <R.Label>{{t (concat 'resources.target.types.' type.type)}}</R.Label>
           <R.Description>{{t
-              (concat 'resources.target.help.' targetType)
+              (concat 'resources.target.help.' type.type)
             }}</R.Description>
-          <R.Icon @name={{icon}} />
+          <R.Icon @name={{type.icon}} />
         </G.RadioCard>
-      {{/each-in}}
+      {{/each}}
     </Hds::Form::RadioCard::Group>
-
+    <br />
```

Comment:
Are we using `<br />` to get some space? if so, can we achieve that with CSS?

Reply:
Its been added to the hds override CSS, and we can get rid of the <br/>